src/utils/isCreatedDoc.js

```diff
@@ -0,0 +1,3 @@
+export default function isCreatedDoc(doc) {
+  return doc._rev.split('-').shift() === '1'
```

Comment:
why not [0] ?

Reply:
I don't know if there is any reason. This code was just copy/pasted from cozy-konnector-libs and left unchanged since it works.